File: Web-Wicket/src/main/java/com/untitleddoc/web/wicket/BasePage.java
package com.untitleddoc.web.wicket;

import lombok.extern.slf4j.Slf4j;
import org.apache.wicket.markup.html.WebPage;
import org.apache.wicket.request.mapper.parameter.PageParameters;

/**
 *
 * @author Novikov
 */
@Slf4j()
public abstract class BasePage extends WebPage {

	private static final long serialVersionUID = -7565557356813846136L;

	public BasePage(final PageParameters parameters) {
	}

}

File: Web-Wicket-Kotlin/src/main/kotlin/com/untitleddoc/cadencecalc/web/wicket/kotlin/BasePage.kt
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.untitleddoc.cadencecalc.web.wicket.kotlin

import org.apache.wicket.markup.html.WebPage
import org.apache.wicket.request.mapper.parameter.PageParameters

/*

  @author kisiy
  Created on 2018/02/12
*/

public abstract class BasePage : WebPage {
    companion object {
        @JvmStatic private val serialVersionUID: Long = -7565557356813846136L
    }

    constructor() : super()
    constructor(parameters: PageParameters) : super(parameters)
}